cmd/nodepool/create.go

```diff
@@ -68,6 +78,9 @@ func NewCreateCommand() *cobra.Command {
 	cmd.Flags().StringVar(&opts.RootVolumeType, "root-volume-type", opts.RootVolumeType, "The type of the root volume (e.g. gp2, io1) for machines in the NodePool")
 	cmd.Flags().Int64Var(&opts.RootVolumeIOPS, "root-volume-iops", opts.RootVolumeIOPS, "The iops of the root volume when specifying type:io1 for machines in the NodePool")
 	cmd.Flags().Int64Var(&opts.RootVolumeSize, "root-volume-size", opts.RootVolumeSize, "The size of the root volume (min: 8) for machines in the NodePool")
+	cmd.Flags().StringVar(&opts.KubevirtMemory, "kubevirt-memory", opts.KubevirtMemory, "In KubeVirt platform - the amount of memory which is visible inside the Guest OS (type BinarySI, e.g. 5Gi, 100Mi)")
+	cmd.Flags().Uint32Var(&opts.KubevirtCPU, "kubevirt-cpu", opts.KubevirtCPU, "In KubeVirt platform - Cores specifies the number of cores inside the vmi, Must be a value greater or equal 1")
+	cmd.Flags().StringVar(&opts.KubevirtImage, "kubevirt-image", opts.KubevirtImage, "In KubeVirt platform - Image is a reference to docker image with the embedded disk to be used to create the machines")
```

Comment:
should this be `MarkFlagRequired`?

Reply:
I see, can only be marked after addressing https://github.com/openshift/hypershift/pull/779/files#r774560839

Reply:
Can't use MarkFlagRequired, because this is also can be called for other platform
Need to refactor this `nodepool/create` to be organized by platform, like done in `cluster/create`